pkg/resources/metrics.go

```diff
@@ -20,6 +20,7 @@ package resources
 import (
 	"context"
 	"fmt"
+	"github.com/integr8ly/integreatly-operator/pkg/config"
```

Comment:
I think I've done same before but maybe let's move it to the bottom/mid section instead